fuzzit.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -ex
+
+# Build fuzzers
+go-fuzz-build -libfuzzer -o uploader-fuzzer.a ./dfget/core/uploader
```

Comment:
So curious about that why only to build fuzzers for `./dfget/core/uploader`  instead of all dfget? And so do with the `./supernode/daemon/mgr/cdn`.

And I think more fuzzing test will be added later.